example/b-flash-message/flash_message.eml.ml

```diff
@@ -0,0 +1,55 @@
+let input_form request =
+  <html>
+    <body>
+      Enter some text:
+      <%s! Dream.form_tag ~action:"/" request %>
+        <input name="text" autofocus>
+      </form>
+    </body>
+  </html>
+
+
```

Comment:
Is mixing Tyxml and templates like this idiomatic? I couldn't figure out how to implement `results_page` using templates.

Reply:
I'll edit this example to not use TyXML after the merge, and ping.